ci/buildkite-solana-private.sh

```diff
@@ -168,7 +168,7 @@ all_test_steps() {
              ^sdk/ \
       ; then
     cat >> "$output_file" <<"EOF"
-  - command: "ci/test-stable-sbf.sh"
+  - command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-wasm.sh"
```

Comment:
is it intentional to switch the test from stable-sbf to wasm here?

Reply:
my bad 🫠 fixed